plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -709,6 +708,20 @@ public void testAddColumnConcurrently()
         throw new SkipException("TODO");
     }
 
+    @Test
+    public void testSetColumnTypeNonLowercaseColumn()
```

Comment:
Can we have a test for changing the type of a nested column?

`row(integer) -> row(biginteger)`

Failure case 

`integer -> row(integer)`